toolchain/static-link.lisp

```diff
@@ -33,6 +33,16 @@
 
 ;; Problem? Its output may conflict with the program-op output :-/
 
+(defmethod perform :before ((op static-program-op) (s system))
```

Comment:
static-program-op inherits from static-image-op. IIUC, your fix is useful for both.

Reply:
Done.